src/services/infra/__tests__/DynamoDBService.spec.ts

```diff
@@ -41,7 +41,7 @@ const mockFailureLaunch: SiteLaunchMessage = {
   },
 }
 
-const tableName = "site-launch"
+const tableName = "ddb-name"
```

Comment:
why do we have a change in table name here?

Reply:
oh this is due to the var in .env.test using `ddb-name`